Lib/test/test_typing.py

```diff
@@ -2654,7 +2654,7 @@ def test_get_type_hints_modules(self):
         self.assertEqual(gth(ann_module2), {})
         self.assertEqual(gth(ann_module3), {})
 
-    @expectedFailure
+    @skip("known bug")
```

Comment:
```suggestion
    @skip("known bug : issue #33758")
```

Can the issue number be added as part of this message ?

Reply:
The test body below has a better explanation, while the issue is more about the "expected failure" being flaky.

Reply:
That doesn't make sense because that's not the issue to fix the bug exposed by the test.

Reply:
Got it, sorry.